plugins/restful/RestfulEntityBase.php

```diff
@@ -324,15 +355,17 @@ public function viewEntity($id) {
         }
 
         if (empty($info['formatter'])) {
-          if ($sub_wrapper instanceof EntityListWrapper) {
-            // Multiple values.
-            foreach ($sub_wrapper as $item_wrapper) {
-              $value[] = $this->getValueFromProperty($wrapper, $item_wrapper, $info, $public_field_name);
+          if (field_is_translatable($this->entityType, field_info_field($info['property']))) {
```

Comment:
By the documentation of `field_info_field`:
>  * @return
 *   The field array, as returned by field_read_fields(), with an
 *   additional element 'bundles', whose value is an array of all the bundles
 *   this field belongs to keyed by entity type. NULL if the field was not
 *   found.

So I guess if this is not a Field API field it returns `NULL`,
hope i'm not talking nonsense...